concordia/templates/transcriptions/campaign_detail.html

```diff
@@ -14,8 +14,22 @@
     <div class="row">
         <div class="col-12">
             <h1 class="my-default ml-3">{{ campaign.title }}</h1>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-9">
             <p class="ml-default mb-default hero-text">{{ campaign.description }}</p>
         </div>
+        <div class="col-3 related-links">
```

Comment:
Since we're using Bootstrap, should we just use something like a [list group](http://getbootstrap.com/docs/4.1/components/list-group/)?